PlayerControls/PlayerControls.xcodeproj/project.pbxproj

```diff
@@ -587,6 +588,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
+				CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = YES;
```

Comment:
Make this `NO`

Reply:
Or instead this setting ignores localization warnings?

Reply:
Exactly